app/scripts/init-globals.js

```diff
@@ -0,0 +1,18 @@
+/**
+ * This script add properties in globalThis and initialises them with undefined.
+ * This is workaround needed to avoid error in runtime-cjs.js which is thrown when loading
```

Comment:
error is not in runtime-cjs, error is in dependencies expecting to be run in a browser window and not a worker

Reply:
This is changed now.